src/markup-components/component-wrapper.tsx

```diff
@@ -1,18 +1,18 @@
 // Copyright © 2023 Ory Corp
 // SPDX-License-Identifier: Apache-2.0
 
-import { FunctionComponent } from "react"
+import { FunctionComponent, PropsWithChildren } from "react"
 import ReactDOMServer from "react-dom/server"
 import { IntlProvider } from "../react-components"
 import * as locales from "../locales"
 
-export type Context = {
+export interface Context {
   locale?: keyof typeof locales
 }
 
-export const ComponentWrapper = <Props extends {}>(
-  Component: FunctionComponent<Props>,
-  props: Props,
+export const ComponentWrapper = <WrapperProps extends PropsWithChildren>(
```

Comment:
eslint strict types don't like the usage of a type extending an empty object (any).

Not sure if this is the correct solution to this. Another option might be to exclude this file from eslint?